endpoints/cookie_sync.go

```diff
@@ -108,6 +109,9 @@ func (deps *cookieSyncDeps) Endpoint(w http.ResponseWriter, r *http.Request, _ h
 
 	parsedReq.filterExistingSyncs(deps.syncers, userSyncCookie)
 	parsedReq.filterForGDPR(deps.syncPermissions)
+	if parsedReq.Limit > 0 {
```

Comment:
Since `if req.Limit <= 0` is checked inside `filterToLimit()`, you don't need to check `> 0` here.

Reply:
Yes, I was going to remove that when I moved the check, then forgot to go back and actually remove it. Thanks.